core/point/point.go

```diff
@@ -1,13 +1,40 @@
 // Package point is a basic package for points.
 package point
 
+import (
+	"fmt"
+)
+
 // Point is a basic point. Although simple, the member variables are kept
 // private to ensure that Point remains immutable.
 type Point struct {
 	x int64
 	y int64
 }
 
+// pointToSgfMap is a translation reference between int64 Point
+// and string SGF-Point (rune) values
+var pointToSgfMap = map[int64]rune{
```

Comment:
You shouldn't have this diff. I think you need to

```
git fetch upstream
git merge upstream/master
```

If you don't have upstream set up, you'll need to

```
git remote add upstream git@github.com:otrego/clamshell.git
```